api/pom.xml

```diff
@@ -43,6 +43,24 @@
             <version>1.1.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>biz.aQute.bnd</groupId>
+            <artifactId>biz.aQute.bnd.annotation</artifactId>
+            <version>${bnd.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.annotation.bundle</artifactId>
```

Comment:
Why not use `osgi.annotation;version-7.0.0`?

Reply:
And we probably should hoist this up into the parent pom since the javadoc goal for the tck project complains about missing types from this artifact.

Reply:
I was just following the pattern since there was already a dep for `org.osgi.annotation.versioning`. I'd be fine with switching to `osgi.annotation`.

The parent pom doesn't buy much since it's only for this one spec anyway. There is no uber-parent pom for all specs that I can see.

Reply:
but yeah. to solve the javadoc we could hoist it...